src/renderer/components/layout/main-layout-header.tsx

```diff
@@ -1,33 +1,34 @@
-import { observer } from "mobx-react";
 import React from "react";
+import { observer } from "mobx-react";
 
-import { clusterSettingsURL } from "../+cluster-settings";
-import { broadcastMessage } from "../../../common/ipc";
+import { ClusterActions } from "../cluster-manager";
 import { Cluster } from "../../../main/cluster";
 import { cssNames } from "../../utils";
-import { Icon } from "../icon";
+import { MenuActions, MenuItem } from "../menu";
 
 interface Props {
   cluster: Cluster
   className?: string
 }
 
 export const MainLayoutHeader = observer(({ cluster, className }: Props) => {
+  const actions = ClusterActions(cluster);
+  const renderMenu = () => <MenuActions autoCloseOnSelect className="ClusterActionsMenu">
```

Comment:
nit: I find starting the root tag on a new line, slightly easier to read:
```suggestion
  const renderMenu = () => (
    <MenuActions autoCloseOnSelect className="ClusterActionsMenu">
```